recipe/meta.yaml

```diff
@@ -33,13 +31,13 @@ requirements:
     - python
     - pip
     - setuptools
-    - llvmlite 0.31.*
+    - llvmlite 0.31.0
     - numpy
-    - tbb-devel >=2018.0.5  # [linux]
+    - tbb-devel >=2019.5  # [linux]
 
   run:
     - python
-    - {{ pin_compatible('llvmlite', max_pin='x.x') }}
+    - llvmlite >=0.31.0,<0.33
```

Comment:
It seems like there should be a better way to express this.

Reply:
This was needed. https://github.com/numba/numba/blob/release0.49/buildscripts/condarecipe.local/meta.yaml#L35
```
 - llvmlite 0.31.*|0.32.*
```

Reply:
@jjhelmus, do you have thoughts on this?